extensions/BMO/Extension.pm

```diff
@@ -715,13 +715,13 @@ sub bug_format_comment {
 
     # link github pull requests and issues
     push (@$regexes, {
-        match => qr#\s([A-Za-z0-9_\.-]+)\/([A-Za-z0-9_\.-]+)\#([0-9]+)\b#mx,
+        match => qr/\s([A-Za-z0-9_\.-]+)\/([A-Za-z0-9_\.-]+)\#([0-9]+)\b/,
         replace => sub {
             my $args = shift;
             my $owner = html_quote($args->{matches}->[0]);
             my $repo = html_quote($args->{matches}->[1]);
             my $number = html_quote($args->{matches}->[2]);
-            return qq#<a href="https://github.com/$owner/$repo/issues/$number">$owner/$repo\#$number</a>#;
+            return qq#<a target = "_blank" href="https://github.com/$owner/$repo/issues/$number"> $owner/$repo\#$number</a>#;
```

Comment:
Hmmm, probably you should do the same thing as the svn regexp that, capture `\s` as `[0]` and prepend to the result, instead of adding a whitespace, because `\s` can match other characters as well. Also it doesn't seem to me we usually do `target="_blank"` on links. Even if we do, there shouldn't be whitespaces around `=`.